tests/cli/test_config.py

```diff
@@ -464,8 +464,8 @@ def test_set_unconfigured_key(conda_cli: CondaCLIFixture):
 def test_set_invalid_key(conda_cli: CondaCLIFixture):
     key, to_val = "invalid_key", "a_bogus_value"
     with make_temp_condarc(CONDARC_BASE) as rc:
-        with pytest.raises(CondaKeyError, match=r"'invalid_key': unknown parameter"):
-            conda_cli("config", "--file", rc, "--set", key, to_val)
+        out, err, _ = conda_cli("config", "--file", rc, "--set", key, to_val)
+        assert err == f"Unknown key: '{key}'\n"
```

Comment:
So an exception is no longer raised, does this mean the exit code is no longer non-zero?

Reply:
Yes, I changed it to no longer raise and exception and return error code, and I will change it back to the way it was.

The reason I changed it originally is that our handling of error codes in `conda config` is inconsistent. For example, `conda config --get does_not_exist` does not return a non-zero error code when it probably should.